project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.1.1
+sbt.version=1.2.6
```

Comment:
Does this have any larger implications? Is this necessary for this PR?

Reply:
This was just me attempting to work around sbt being weird with the snapshots. I think the 1.1 sbt line has a bug with preferring published snapshots to local things. This should be removed eventually, but may be needed to test against the firrtl PR.